### ISSUE TITLE
Bump humantime 2.1.0

### ISSUE THREAD

**Issue by dongxuwang** [jean-airoldie/humantime-serde#10]

**Comment by jean-airoldie** [comment]
I'm unsure what different it makes because minor bumps are supposed to be non-breaking, so from the user's perspective a "2.0" dependency is strictly better than a "2.1" dependancy. Am I missing something?

**Comment by Enyium** [comment]
> Am I missing something?

I don't think so. Cargo should only see a roadblock when the next version has a change in the major version number. I think this can be closed.
